### ISSUE TITLE
Support username passed as a parameter in URI

### ISSUE THREAD

**Issue by ernestrc** [prestodb/presto#4464]
I'm setting the user property in the URL: jdbc:presto://xxxxxx.us-west-2.compute.amazonaws.com:8889?user=sonicd, but driver still complains:

```
Error: java.sql.SQLException: Username property (user) must be set
    at com.facebook.presto.jdbc.PrestoDriver.connect(PrestoDriver.java:86)
    at java.sql.DriverManager.getConnection(DriverManager.java:664)
    at java.sql.DriverManager.getConnection(DriverManager.java:270)
    at build.unstable.sonic.source.JdbcPublisher$$anonfun$receive$1.applyOrElse(JdbcSource.scala:129)
    at akka.actor.Actor$class.aroundReceive(Actor.scala:480)
    at build.unstable.sonic.source.JdbcPublisher.akka$stream$actor$ActorPublisher$$super$aroundReceive(JdbcSource.scala:33)
    at akka.stream.actor.ActorPublisher$class.aroundReceive(ActorPublisher.scala:270)
    at build.unstable.sonic.source.JdbcPublisher.aroundReceive(JdbcSource.scala:33)
    at akka.actor.ActorCell.receiveMessage(ActorCell.scala:526)
    at akka.actor.ActorCell.invoke(ActorCell.scala:495)
    at akka.dispatch.Mailbox.processMailbox(Mailbox.scala:257)
    at akka.dispatch.Mailbox.run(Mailbox.scala:224)
    at akka.dispatch.Mailbox.exec(Mailbox.scala:234)
    at scala.concurrent.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260)
    at scala.concurrent.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1339)
    at scala.concurrent.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979)
    at scala.concurrent.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107)
```

Not sure the check on `at com.facebook.presto.jdbc.PrestoDriver.connect(PrestoDriver.java:86)` is correct, given that username can be passed in the url:

``` java
@Override
    public Connection connect(String url, Properties info)
            throws SQLException
    {
        if (!acceptsURL(url)) {
            return null;
        }

        String user = info.getProperty(USER_PROPERTY);
        if (isNullOrEmpty(user)) {
            throw new SQLException(format("Username property (%s) must be set", USER_PROPERTY));
        }

        return new PrestoConnection(parseDriverUrl(url), user, queryExecutor);
    }
```

**Comment by electrum** [comment]
This is not actually a bug. JDBC does have any requirements for the URL aside from the generic format `jdbc:<subprotocol>:<subname>`. The standard way to provide a username is the `getConnection` method on `DriverManager`, which then passes it to the `Driver` via `Properties` passed to the `connect` method.

That said, we could certainly add support for specifying the username as part of the URL (as most other drivers allow).

**Comment by jirassimok** [comment]
This seems to work now. Query parameters have been included in the connection properties since defabcba4e. It's also included in the [documentation for the JDBC driver](https://prestodb.io/docs/current/installation/jdbc.html) (since 9cc3803861a).
